scripts/workers/submission_worker.py

```diff
@@ -130,6 +130,35 @@ def download_and_extract_file(url, download_location):
                     f.write(chunk)
 
 
+def extract_zip_file(download_location, extract_location):
```

Comment:
@nikochiko this is a great work on modularizing the code for submission worker. But it would be good to have this in a separate PR. We can create a GCI task for the same. Can you please remove these changes from this PR and open a new PR

Reply:
Sure! Will do that.

Reply:
@Ram81 The new PR: https://github.com/Cloud-CV/EvalAI/pull/2532 
Please have a look.

Reply:
@nikochiko we can revert these changes in this PR, as #2532 has the same change.

Reply:
@pushkalkatara the tests written are dependent on these functions 😕

Reply:
Okay I think we can afford the Travis build failing for a few hours. Was worried for no reason, will make the changes :+1: